app/routes/items/new.js

```diff
@@ -73,10 +83,31 @@ export default AuthorizeRoute.extend({
         controller.set("length", null);
         controller.set("width", null);
         controller.set("height", null);
-        controller.set("selectedGrade", { name: "B", id: "B" });
-        controller.set("selectedCondition", { name: "Used", id: "U" });
+        controller.set("selectedGrade", {
+          name: "B",
+          id: "B"
+        });
+        controller.set("selectedCondition", {
```

Comment:
Careful for the conditions, a while ago we replaced conditions by the actual lookup table
```javascript
conditions: Ember.computed(function() {
    return this.get("store").peekAll("donor_condition");
  })
```
So id `"U"` wouldn't match anything anymore